pact/coin-contract/coin.pact

```diff
@@ -184,9 +194,9 @@
     @doc "Create an account for ACCOUNT, with GUARD controlling access to the  \
     \account."
 
-    @model [ (property (account-structure account)) ]
+    @model [ (property (account account)) ]
```

Comment:
this might be a good one to put at module level (ie list the functions that must cover this), see https://github.com/kadena-io/pact/blob/master/examples/verified-accounts/accounts.pact#L22

Reply:
Wow i didn't even know that was a feature!